scripts/sct_pipeline.py

```diff
@@ -52,9 +52,10 @@
 import copy
 from time import time, strftime
 if "SCT_MPI_MODE" in os.environ:
-    from distribute2mpi import MpiPool as Pool
+    from mpi4py.futures import MPIPoolExecutor as PoolExecutor
 else:
-    from multiprocessing import Pool
+    # from multiprocessing import Pool
```

Comment:
You can remove the comment. It might confuse readers that the import might be important.